src/test/java/placement/model/BoardTests.java

```diff
@@ -0,0 +1,338 @@
+    }
+
+    @Test
+    public void givenNewlyCreatedBoard_WhenShipIsPlacedCorrectly_ItIsNotEmpty() {
+        Board board = Board.createDefaultBoard();
+        Ship ship = new Ship(4, Direction.SOUTH);
+
+        board.placeShip(ship, 8, 3);
+
+        Assert.assertFalse(board.isEmpty());
+    }
+
+    @Test
+    public void givenNewlyCreatedBoard_WhenShipIsPlacedIncorrectly_ItIsEmpty() {
+        Board board = Board.createDefaultBoard();
+        Ship incorrectShip = new Ship(3, Direction.NORTH);
+
+        board.placeShip(incorrectShip, 5, 15);
+
+        Assert.assertTrue(board.isEmpty());
+    }
+
+    @Test
+    public void givenBoard_WhenShipWithOneMastIsPlaced_ThenFieldIsMarkedAsOccupied() {
+        Board board = Board.createDefaultBoard();
+        Ship ship = new Ship(1, Direction.SOUTH);
+
+        board.placeShip(ship, 6, 6);
+
+        Assert.assertTrue(board.isFieldOccupied(6, 6));
+        Assert.assertFalse(board.isFieldOccupied(3, 7));
+    }
+
+    @Test
+    public void givenEmptyBoard_WhenShipWithLengthIsPlaced_ThenAllFieldAreMarkedAsOccupied() {
+        Board board = Board.createDefaultBoard();
+        Ship ship = new Ship(3, Direction.EAST);
+
+        board.placeShip(ship, 5, 5);
+
+        Assert.assertTrue(board.isFieldOccupied(5, 5));
+        Assert.assertTrue(board.isFieldOccupied(6, 5));
+        Assert.assertTrue(board.isFieldOccupied(7, 5));
+
+        Assert.assertFalse(board.isFieldOccupied(8, 5));
+        Assert.assertFalse(board.isFieldOccupied(4, 5));
+    }
+
+    @Test
+    public void givenEmptyBoard_WhenShipIsPlaced_ThenSurroundingFieldsAreBuffers() {
+        Board board = Board.createDefaultBoard();
+        Ship ship = new Ship(1, Direction.NORTH);
+        board.placeShip(ship, 3, 3);
+
+        // (3, 3) is not a buffer, but a ship
+        Assert.assertFalse(board.isFieldBuffer(3, 3));
+
+        // 8 surrounding fields are supposed to be buffers now
+        Assert.assertTrue(board.isFieldBuffer(2, 3));
+        Assert.assertTrue(board.isFieldBuffer(4, 3));
+        Assert.assertTrue(board.isFieldBuffer(2, 2));
+        Assert.assertTrue(board.isFieldBuffer(3, 2));
+        Assert.assertTrue(board.isFieldBuffer(4, 2));
+        Assert.assertTrue(board.isFieldBuffer(2, 4));
+        Assert.assertTrue(board.isFieldBuffer(3, 4));
+        Assert.assertTrue(board.isFieldBuffer(4, 4));
+
+        // just a quick check that some other field is not a buffer (nor occupied)
+        Assert.assertFalse(board.isFieldBuffer(5, 5));
+        Assert.assertFalse(board.isFieldBuffer(2, 1));
+    }
+
+    @Test
+    public void givenEmptyBoard_WhenShipIsPlacedInCorner_ThenSurroundingFieldsAreBuffers() {
+        Board board = Board.createDefaultBoard();
+        Ship ship = new Ship(1, Direction.NORTH);
+        board.placeShip(ship, 0, 0);
+
+        // (3, 3) is not a buffer, but a ship
+        Assert.assertFalse(board.isFieldBuffer(0, 0));
+
+        // 8 surrounding fields are supposed to be buffers now
```

Comment:
// 8 surrounding fields are supposed to be buffers now 
but why only 3 of them?